cmd/ingest/cmd/root.go

```diff
@@ -16,14 +16,64 @@
 package cmd
 
 import (
+	"context"
 	"fmt"
 	"os"
+	"strings"
 
+	"github.com/guacsec/guac/pkg/logging"
+
+	homedir "github.com/mitchellh/go-homedir"
 	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
 )
 
+var cfgFile string
+
 func init() {
-	rootCmd.AddCommand(exampleCmd)
+	cobra.OnInitialize(initConfig)
+	persistentFlags := rootCmd.PersistentFlags()
+	persistentFlags.StringVar(&flags.dbAddr, "db-addr", "neo4j://localhost:7687", "address to neo4j db")
```

Comment:
I'm thinking of making `db-addr` be `gdbaddr` to match with `gdbuser` and `gdbpass`. This will also eliminate the need to replace`-` with `_` in the envvar

Reply:
done!